app/NewOrder.php

```diff
@@ -170,39 +170,39 @@ public function getOrderPaymentMethodAttribute()
         })->first()->paymentMethod->name ?? null;
     }
 
-    public function toArray()
-    {
-        return [
-            "id" => $this->id,
-            "order_number" => $this->order_number,
-            "product_id" => $this->product_id,
-            "product" => $this->product,
-            "product_name" => $this->product->name,
-            "serial_number" => $this->serial_number,
-            "repayment_duration" => $this->repaymentDuration->name ?? null,
-            "repayment_cycle" => $this->repaymentCycle->name ?? null,
-            "customer_id" => $this->customer->id,
-            "customer_name" => $this->customer->fullName,
-            "customer_phone" => $this->customer->telephone,
-            "customer_email" => $this->customer->email,
-            "business_type" => $this->businessType->name ?? null,
-            "status" => $this->orderStatus->name ?? null,
-            "branch" => $this->branch->name,
-            "product_price" => $this->product_price,
-            "down_payment" => $this->down_payment,
-            "repayment" => $this->repayment,
-            "discount" => $this->discounts,
-            "single_repayment" => $this->amortization[0]->expected_amount ?? '',
-            "custom_date" => $this->customDate->custom_date ?? null,
-            "amortization" => $this->amortization,
-            "notifications" => $this->notifications,
-            "order_payment_method" => $this->order_payment_method,
-            "customer" => $this->customer,
-            "order_date" => $this->order_date,
-            "owner" => $this->owner->full_name ?? '',
-            "sales_type" => $this->salesCategory ?? '',
-            "branch_id" => $this->branch->id,
-            "owner_id" => $this->owner->id
-        ];
-    }
+    // public function toArray()
```

Comment:
Why did you comment this out?

Reply:
will be cleaned, did that while I was debugging